docs: Add tuts for Bytebase SaaS and Amazon Aurora (#884)
bytebase/bytebase.com: content/blog/database-change-management-using-bytebase-saas.md

---

> title: Database Change Management using Bytebase SaaS

Reviewer: Bytebase SaaS -> Bytebase Cloud

SaaS -> Cloud

All occurrences as well as the page url

author: Ningjing
published_at: 2023/04/17 16:15
feature_image: /static/blog/database-change-management-using-bytebase-saas/feature-image.webp
tags: Tutorial
integrations: MySQL
level: Beginner
description: Bytebase provide its SaaS version since 1.15.0, this tutorial is a step-by-step guide to set up Database Change Management for Amazon Aurora MySQL using Bytebase SaaS. 
---
This is a series of articles about Database Change Management using Bytebase SaaS. We take Amazon Aurora MySQL as an example and is also applicable to Amazon Aurora Serverless.

- Database Change Management using Bytebase SaaS (this one)
- [Database Change Management with GitHub using Bytebase SaaS](/blog/database-change-management-with-github-using-bytebase-saas)

---

> Bytebase is an open-source database CI/CD tool for developers and DBAs. It provides its SaaS version since [1.15.0](/changelog/bytebase-1-15-0). You can still use its [self-hosted version](/docs/get-started/install/deploy-with-docker).

Reviewer: ```suggestion
Bytebase is an open-source database CI/CD tool for developers and DBAs. It provides its SaaS version since [1.15.0](/changelog/bytebase-1-15-0) in addition to the existing [self-hosted version](/docs/get-started/install/deploy-with-docker).
```
